discord_slash/utils/manage_commands.py

```diff
@@ -235,3 +313,38 @@ def create_choice(value: str, name: str):
         "value": value,
         "name": name
     }
+
+
+def create_permission(id: typing.Union[int], id_type: int, permission: bool):
+    if not isinstance(id_type, int) or isinstance(id_type, bool): #Bool values are a subclass of int
+        original_type = id_type
+        id_type = SlashCommandPermissionsType.from_type(original_type)
+        if id_type is None:
+            raise IncorrectType(f"The type {original_type} is not recognized as a type that Discord accepts for slash command permissions.")
+    return {
+        "id": str(id),
```

Comment:
```suggestion
        "id": id,
```
Snowfakes are integers